app/application/views/onboarding_3_enter_details_form.php

```diff
@@ -54,13 +65,22 @@
                                     <form action="<?php echo site_url('/onboard/send_details'); ?>" method="post">
 
                                         <div class="form-group">
-                                            <label for="exampleInputEmail1">First Name</label>
```

Comment:
Please could you explain what changes you've made to this on-boarding page. Just want to make sure we don't conflict across branches.

Reply:
didnt make any changes - what happened was id branched off from main before id pulled in the changes to that page, and merely copy pasted it in from master so that i could have syntax highlighting and see the relevant changes needed to be made to my volunteering and with select2 in general, you can ignore this change etc...